build.gradle

```diff
@@ -51,7 +51,7 @@ subprojects {
         ]
         if (project.path.startsWith(":compiler")) {
             extraKotlincArgs += [
-                "-opt-in=motif.compiler.processing.ExperimentalProcessingApi"
+                "-opt-in=androidx.room.compiler.processing.ExperimentalProcessingApi"
```

Comment:
Speaking from our intuition, `androidx.room` has nothing to with `motif`. The reason why we have to refer `androidx.room` here is because we need to leverage the common code such as  `androidx.room.compiler.processing.XAnnotation` by following the path in `XProcessing `.

Let me know if I totally misinterpreted the reason.

Reply:
Yeah the XProcessing library is under the androidx.room library group. I was initially going to jarjar it to a new package but I've punted on that for now since I ran into an issue was extension functions